tools/db_stress.cc

```diff
@@ -2810,7 +2810,10 @@ class BatchedOpsStressTest : public StressTest {
       WriteOptions& /* write_opts */,
       const std::vector<int>& /* rand_column_families */,
       const std::vector<int64_t>& /* rand_keys */,
-      std::unique_ptr<MutexLock>& /* lock */) { assert(false); }
+      std::unique_ptr<MutexLock>& /* lock */) {
+    assert(false);
+    return Status::OK();
```

Comment:
does it sound a bit self contradictory that this function asserts and returns OK status?

Reply:
Done. Will push shortly.